### DISCUSSION TITLE
Scintilla: restrict the horizontal fold line to the visible span of the header text line

### DISCUSSION THREAD

**Discussion by ghost** [geany/geany#2932] [Ideas]
Anyone else finds those fold lines too long ? 
I know that the fold lines are done by Scintilla, so I submitted a feature request there: https://sourceforge.net/p/scintilla/feature-requests/1419/ 
With attached pictures to see the proposal at a glance.
If you like it, or have suggestions,  you can comment there .

**Comment by eht16** [comment]
If Scintilla supports it and provides an API to configure this feature, we can add an option in Geany. But we won't add it directly in Geany.
